packages/components/organisms/f-checkout/CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+v0.56.1
+-------------------------------
+*_February 11, 2021_*
```

Comment:
Nit: Would you mind going back for the last few versions and change `*_date_*` to `*date*`? It seems to have snuck in at some point and isn't our usual convention.

Reply:
Done